etc/db-config.yaml

```diff
@@ -137,6 +137,11 @@
             default_value: true
             public: false
             description: Allows to distribute one judging to multiple judgehosts.
+        -   name: judgehost_enabled_by_default
+            type: bool
+            default_value: true
+            public: false
+            description: Enable a judgehost by default when a new one connects.
```

Comment:
We call this "active" in the UI I think. Although I could see a case that that term is the thing we should change instead of this

Reply:
For now i will make it consistently be `active`. we can change later.